tests/snmp/test_snmp_phy_entity.py

```diff
@@ -615,7 +615,7 @@ def test_turn_off_psu_and_check_psu_info(duthosts, enum_rand_one_per_hwsku_hostn
     pdu_controller.turn_off_outlet(first_outlet)
     assert wait_until(30, 5, check_outlet_status, pdu_controller, first_outlet, False)
     # wait for psud update the database
-    assert wait_until(120, 20, _check_psu_status_after_power_off, duthost, localhost, creds_all_duts)
+    assert wait_until(180, 20, _check_psu_status_after_power_off, duthost, localhost, creds_all_duts)
```

Comment:
The magic number seem fragile. If it is related to port number, could we wait for some deterministic port event?

Reply:
Qi, I agree with you in principal.

But wait_until can use some larger wait time, our expectation is that test should mostly passing and when test passes, the wait time would be much lower than the threshold. The number dictates how long do we wait until failing the testcase. So in this line, I am ok with a more generous wait time.